handbook/communication/rfcs/index.md

```diff
@@ -1,139 +1,173 @@
-# Requests for comments (RFCs)
+# Requests for Comments (RFCs)
 
-We value writing down plans so that we can asynchronously communicate to and solicit feedback from our [all-remote](../../company/remote/index.md) team. A good plan communicates what problem is being solved, why that problem is being prioritized now, who will make the final decision(s), and what the plan is to solve the identified problem.
-
-This document describes how we operationalize written planning through our RFC process. RFC literally means "Request for Comments" and you can think about it as exactly that, no more, no less.
-
-This process is designed to be lightweight so that it can be used for many purposes (e.g. product specs, policy decisions, technical discussion), and it is optimized for facilitating collaboration and feedback. In contrast, GitHub issues are best for tracking concrete bug reports or work that has already been scoped and planned (i.e. there isn't much remaining to discuss).
-
-We encourage you to understand [how decisions are made at Sourcegraph](../decisions.md) before writing an RFC.
+We use RFCs to collaborate successfully as an [all-remote](../../company/remote/index.md) team.
```

Comment:
I don't know where exactly to fit this in, but I think it might be valuable to have a quickstart section a the top that let's someone, who is already familiar with RFCS, quickly create a new RFC.

```markdown
## Quickstart

* [Folder of public RFCs](#link)
* [Folder of private RFCs](#link)
* Create a new RFC
  * [Default format](https://docs.google.com/document/d/1VV0ddLmMrcU2IWo_s4xm8Q8UtGnreyuhIRPJY0Dh5NI/copy)
  * (list other formats)
```

 (note that you can have a google doc link end with `/copy` to prompt the user to make their own copy)

Reply:
Added!

Thanks for the tip on `/copy` – I had no idea!